besspin/cyberPhys/infotainmentserver.py

```diff
@@ -53,8 +53,7 @@ def restart (xTarget):
         # the kill_listeners.sh script
         #xTarget.runCommand("pkill infotainment_se",exitOnError=False,tee=appLog)
         xTarget.runCommand("/opt/kill_listeners.sh",exitOnError=False,tee=appLog)
-        xTarget.runCommand("systemctl stop infotainment-server.service", erroneousContents=["Failed to stop", "error code"], tee=appLog)
-        xTarget.runCommand("systemctl start infotainment-server.service", erroneousContents=["Failed to start", "error code"], tee=appLog)
+        xTarget.runCommand("systemctl restart infotainment-server.service", erroneousContents=["Failed", "error code"], tee=appLog)
```

Comment:
I realize we don't care much about FreeBSD, but shouldn't we change it to `restart` there too?

Reply:
Fixed